src/tls_wrap.cc

```diff
@@ -120,20 +117,18 @@ TLSWrap::~TLSWrap() {
 
 
 void TLSWrap::MakePending() {
-  write_item_queue_.MoveBack(&pending_write_items_);
```

Comment:
You can remove MoveBack() from util.h and util-inl.h now, this is the only place that uses it.